lib/core/src/lib/datatable/components/datatable-cell/datatable-cell.component.ts

```diff
@@ -108,6 +93,24 @@ export class DataTableCellComponent implements OnInit, OnDestroy {
         }
     }
 
+    protected subscribeToRowUpdates() {
```

Comment:
why it's protected? shouldn't be private?

Reply:
Yes, I should be, thx